person_capture/gui_app.py

```
@@ -5859,6 +6308,57 @@ def _build_ui(self):
+        self.hdr_sdr_tonemap_combo.currentIndexChanged.connect(self._on_ui_change)
+
+        self.hdr_sdr_gamut_combo = QtWidgets.QComboBox()
+        for _label, _data in (("Perceptual", "perceptual"), ("Relative", "relative"), ("Saturation", "saturation"), ("Clip", "clip")):
+            self.hdr_sdr_gamut_combo.addItem(_label, _data)
+        _hdrgm = str(getattr(self.cfg, "hdr_sdr_gamut_mapping", "perceptual") or "perceptual")
+        _hdrgm_idx = self.hdr_sdr_gamut_combo.findData(_hdrgm)
+        self.hdr_sdr_gamut_combo.setCurrentIndex(_hdrgm_idx if _hdrgm_idx >= 0 else 0)
+        self.hdr_sdr_gamut_combo.setToolTip("Gamut mapping for full-res HDR->SDR screencap export.")
+        self.hdr_sdr_gamut_combo.currentIndexChanged.connect(self._on_ui_change)
+
+        self.hdr_sdr_contrast_spin = _mk_fspin(0.0, 2.0, 0.05, 2, "float: hdr_sdr_contrast_recovery")
```

Ruff (0.15.11)

[error] Undefined name `_mk_fspin`

(F821)